example/multiple-modals-example.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
```

Comment:
This is a new document in the `example` folder to demonstrate the new capabilities